Gruntfile.js

```diff
@@ -1,47 +1,67 @@
 module.exports = function(grunt) {
-  grunt.loadNpmTasks('grunt-contrib-concat');
-  grunt.loadNpmTasks('grunt-contrib-uglify');
-  grunt.loadNpmTasks('grunt-karma');
 
-  var sourceFiles = grunt.file.readJSON('build.json');
-  grunt.initConfig({
-    concat: {
-      pointerevents: {
-        options: {
-          stripBanners: true,
-          banner: grunt.file.read('LICENSE.txt')
-        },
-        nonull: true,
-        src: sourceFiles,
-        dest: 'pointerevents.dev.js'
-      }
-    },
-    uglify: {
-      pointerevents: {
-        options: {
-          // sourceMap: 'pointerevents.min.js.map',
-          banner: grunt.file.read('LICENSE.txt')
-        },
-        nonull: true,
-        dest: 'pointerevents.min.js',
-        src: sourceFiles
-      }
-    },
-    karma: {
+grunt.loadNpmTasks('grunt-contrib-uglify');
+grunt.loadNpmTasks('grunt-karma');
+
+var version = require('./package').version;
+var header =
+  '/*!\n' +
+  ' * PEP v' + version + ' | https://github.com/jquery/PEP\n' +
+  ' * Copyright jQuery Foundation and other contributors | http://jquery.org/license\n'+
+  ' */\n';
+
+grunt.initConfig({
+  uglify: {
+    pointerevents: {
       options: {
-        configFile: 'karma.conf.js',
-        keepalive: true
+        preserveComments: 'some'
       },
-      pointerevents: {
-      },
-      buildbot: {
-        reporters: 'crbot',
-        logLevel: 'OFF'
-      }
+      nonull: true,
+      dest: 'dist/PEP.min.js',
+      src: 'dist/PEP.js'
     }
-  });
+  },
+  karma: {
+    options: {
+      configFile: 'karma.conf.js',
+      keepalive: true
+    },
+    pointerevents: {
+    },
+    buildbot: {
+      reporters: 'crbot',
+      logLevel: 'OFF'
+    }
+  }
+});
+
+grunt.registerTask('build', function() {
+  var esperanto = require('esperanto');
+  var done = this.async();
+
+  grunt.log.write('Building PEP...');
+  esperanto.bundle({
+    base: 'src',
+    entry: '../pointerevents.js'
+  }).then(function (bundle) {
+    var umd = bundle.toUmd({
+      name: 'PointerEventsPolyfill'
+    });
+    grunt.file.write('dist/PEP.js', header + umd.code);
+  }).then(
+    function() {
+      grunt.log.ok();
+      done();
+    },
+    function(error) {
+      grunt.log.error();
+      done(error);
+    }
+  );
+});
```

Comment:
I like to use modules for simplicity over a bigger Gruntfile. To guarantee it will work, we can lock the modules versions, as we already do in QUnit and other projects.

That grunt plugin is indeed missing some crucial features but I already sent a PR.